eisbuk-admin/src/components/atoms/AttendanceCard/AttendanceCard.tsx

```diff
@@ -36,28 +55,65 @@ const AttendanceCard: React.FC<Props> = ({
   const endTime = luxonEnd.toISOTime().substring(0, 5);
 
   const timeString = `${startTime} - ${endTime}`;
+
   return (
-    <div>
-      <div data-testid="time-string">{timeString}</div>
-      <div>{categories}</div>
-      {userBookings.map((user) => {
-        const isAbsent = absentees?.includes(user.customer_id) || false;
-        return (
-          <div key={user.customer_id}>
-            <div>{user.name}</div>
-            <button
-              type="button"
-              onClick={() =>
-                dispatch(markAttendance(user.customer_id, isAbsent))
+    <Container maxWidth="sm">
+      <List className={classes.root}>
+        <div key={id + "-" + timeString} className={classes.slotWrapper}>
+          <ListItem className={classes.listHeader}>
+            <ListItemText
+              primary={
+                <span>
+                  {timeString} <b>({userBookings.length})</b>
+                </span>
               }
-            >
-              {isAbsent ? "👎" : "👍"}
-            </button>
-          </div>
-        );
-      })}
-    </div>
+              secondary={translateAndJoinTags(categories, type)}
+            />
+          </ListItem>
+          {userBookings.map((user) => {
+            const isAbsent = absentees?.includes(user.customer_id);
+
+            return (
+              <UserAttendance
+                key={`${id}-${user.customer_id}`}
+                isAbsent={isAbsent!}
+                userBooking={user}
+              ></UserAttendance>
+            );
+          })}
+        </div>
+      </List>
+    </Container>
   );
 };
 
+// #region Start Region Local Utils
+
+const translateAndJoinTags = (categories: Category[], type: SlotType) => {
+  const translatedCategories = categories.map((category) =>
+    i18n.t(`Categories.${category}`)
+  );
+  const translatedType = i18n.t(`SlotTypes.${type}`);
+
+  return `${[...translatedCategories, translatedType].join(" ")}`;
+};
+// #endregion Local Utils
+
+// #region Styles
+const useStyles = makeStyles((theme: ETheme) => ({
+  root: {},
+  listHeader: {
+    backgroundColor: theme.palette.primary.light,
+  },
+  slotWrapper: {
+    marginBottom: theme.spacing(1.5),
+    borderWidth: 1,
+    borderStyle: "solid",
+    borderColor: theme.palette.primary.main,
+  },
```

Comment:
To be semantically correct, maybe name this `cardWrapper` or better you simply `wrapper`.